src/lib/bootloader/serial_console.rb

```diff
@@ -0,0 +1,70 @@
+require "yast"
+
+Yast.import "Arch"
+
+module Bootloader
+  class SerialConsole
+    PARITY_MAP = {
+      "n" => "no",
+      "o" => "odd",
+      "e" => "even"
+    }
+    SPEED_DEFAULT = 9600
+    PARITY_DEFAULT = "no"
+    WORD_DEFAULT = ""
+
+    def self.load_from_kernel_args(kernel_params)
+      console_parameter = kernel_params.parameter("console")
+      return nil unless console_parameter
+
+      console_parameter = [console_parameter] unless console_parameter.is_a? ::Array
+      serial_console = console_parameter.find { |p| p =~ /ttyS/ || p =~ /ttyAMA/ }
+      return nil unless serial_console
+
+      console_regexp = /(ttyS|ttyAMA)[[:alpha:]]+([[:digit:]]*),?([[:digit:]]*)([noe]*)([[:digit:]]*)/
```

Comment:
And that constant should have

1. Description - what is is good for
2. Example - what it matches :)